include/seqan3/search/fm_index/bi_fm_index_cursor.hpp

```diff
@@ -1022,6 +1022,28 @@ class bi_fm_index_cursor
                    return locate_result_value_type{sequence_rank-1, sequence_position};
                });
     }
+
+    /*!\cond DEV
+     * \brief Serialisation support function.
+     * \tparam archive_t Type of `archive`; must satisfy seqan3::cereal_archive.
+     * \param archive The archive being serialised from/to.
+     *
+     * \attention These functions are never called directly, see \ref serialisation for more details.
+     */
+    template <cereal_archive archive_t>
+    void CEREAL_SERIALIZE_FUNCTION_NAME(archive_t & archive)
+    {
```

Comment:
Since this function is not visible to the user, we should mention this in the class documentation. I think we might just used `\imlements cerealisable` or something similar.. Can you check other classes?

Also Can you add a changelog entry, since this seems to be of interest to users :)

Reply:
Yes, but then I would also rebase this against the release branch?

Reply:
> 
> 
> Since this function is not visible to the user, we should mention this in the class documentation. I think we might just used `\imlements cerealisable` or something similar.. Can you check other classes?

Yes, it would be great to see directly in the documentation which classes can be serialized with cereal, I had to study the code to see that neither cerealisation is possible, nor I can access the members on my own. I guess adding something like `\implements cerealisable` would be perfectly suitable for this.

Reply:
Currently the documentation is only reflected in the inheritance diagram as far as I can see, https://docs.seqan.de/seqan/3-master-user/interfaceseqan3_1_1cerealisable.html

We should rework this